apps/server/src/modules/board/loadtest/readme.md

```diff
@@ -74,3 +74,66 @@ By typeing `export CARL_CORD_PASSWORD=realpassword` the script will not ask you
 ## Todos
 
 - [ ] enable optional parameter course_id
+
+# New Version: Loadtesting
```

Comment:
I always have to smile about the word "new" in documentations or product names, because either there is bound to be a "newer", or the new will grow very old... I would just remove anything outdated, and just describe the new version as the only loadtests.

Reply:
fixed